File: C++/Step9-Math_2 Prime and Geometric/3009.cpp
//
//  3009.cpp
//  Step9-Math_2 Prime and Geometric
//
//  Created by 김예빈 on 2019. 9. 5..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    cin.tie(NULL);
    ios::sync_with_stdio(false);
    
    int rect[3][2];
    int x, y, i, j;
    
    for(i = 0; i < 3; i++)
        for(j = 0; j < 2; j++)
            cin >> rect[i][j];
    
    x = rect[0][0];
    y = rect[0][1];
    x = rect[1][0] == x ? rect[2][0] : (rect[1][0] == rect[2][0] ? x : rect[1][0]);
    y = rect[1][1] == y ? rect[2][1] : (rect[1][1] == rect[2][1] ? y : rect[1][1]);
    
    cout << x << " " << y;

    return 0;
}

File: BaekJoon Online Judge/Swift/Step10-Geometric, Rectangle and Triangle/3009.swift
//
//  3009.swift
//  Step09-Math_2 Prime and Geometric
//
//  Created by Yebin Kim on 2019/11/22.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

var pos = readLine()!.split(separator: " ").map{ Int($0)! }
pos.append(contentsOf: readLine()!.split(separator: " ").map{ Int($0)! })
pos.append(contentsOf: readLine()!.split(separator: " ").map{ Int($0)! })

var setX = Set<Int>()
var setY = Set<Int>()

for i in 0...2 {
    if setX.contains(pos[2 * i]) {
        setX.remove(pos[2 * i])
    } else {
        setX.insert(pos[2 * i])
    }
    
    if setY.contains(pos[2 * i + 1]) {
        setY.remove(pos[2 * i + 1])
    } else {
        setY.insert(pos[2 * i + 1])
    }
}

print("\(setX.first!) \(setY.first!)")
